assets/theme-i18n.json

```
@@ -1,4 +1,5 @@
 {
+	"name": "Name of the style variation",
```

```suggestion
	"name": "Style variation name",
```

This format is more in line with the other contexts

Of course you need to update https://github.com/WordPress/gutenberg/blob/f9c2838a0b8be94fcf4e51b0a831d4629eaf24d7/lib/compat/wordpress-5.9/theme-i18n.json as well.